### ISSUE TITLE
java.lang.RuntimeException: Failed to call observer method

### ISSUE THREAD

**Issue by SomeKoder** [yshrsmz/KeyboardVisibilityEvent#59]
Hello! Love the library, I'm just experiencing a crash while running Espresso tests.
Steps to reproduce:

- Create a test
```kotlin
    @Test
    fun launchFragment(){
        launchFragmentInContainer<MyFragment>()
    }
```

- Create a fragment extension to listen to the keyboard visibility events
```kotlin
fun Fragment.setOnKeyboardShowingListener(then: (isOpen: Boolean)-> Unit){ 
    setEventListener(
        requireActivity(),
        viewLifecycleOwner,
        object : KeyboardVisibilityEventListener {
            override fun onVisibilityChanged(isOpen: Boolean) {
                then.invoke(isOpen)
            }
        })
}
```

- Add this extension to MyFragment
```kotlin
    setOnKeyboardShowingListener {
            if (it){
                // Do something
            }
        }
```

- When espresso tests finishes, the crash seems to occur when destroying the fragment:
```
Caused by: java.lang.IllegalStateException: getContentRoot(activity).getChildAt(0) must not be null
at net.yslibrary.android.keyboardvisibilityevent.KeyboardVisibilityEvent.getActivityRoot$keyboardvisibilityevent_release(KeyboardVisibilityEvent.kt:147)
at net.yslibrary.android.keyboardvisibilityevent.SimpleUnregistrar.unregister(SimpleUnregistrar.kt:30)
at net.yslibrary.android.keyboardvisibilityevent.KeyboardVisibilityEvent$setEventListener$1.onDestroy(KeyboardVisibilityEvent.kt:44)
at java.lang.reflect.Method.invoke(Native Method)
at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:216)
```

The extension is not necessary, it crashes even if I have the setEventListener directly in my fragment.
Again, I love the library, hopefully we can find a solution for this <3

**Comment by yshrsmz** [comment]
possible fix for this issue is released in [3.0.0-RC3](https://github.com/yshrsmz/KeyboardVisibilityEvent/releases/tag/v3.0.0-RC3)

**Comment by VyacheslavDev** [comment]
The app is still not working. 3.0.0-RC3 removes the problem. But the app just close
